zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/socket/Message.java

```diff
@@ -65,6 +65,8 @@
 
     NOTE_RENAME,
 
+    CONVERT_NOTE,     // converting note to nbformat
```

Comment:
I'd extend this to include nbformat, eg. CONVERT_NOTE_NBFORMAT

Reply:
Done